usage/usageplot.py

```diff
@@ -241,11 +247,53 @@ def get_nodehours(times, nodes):
     return node_hours
 
 
+def get_elapsed(start_end, times, jobs):
+    """Finds the elapsed times for each code (ie, how much time it took to
+    complete all coadd jobs, etc.).
+
+    Parameters
+    ----------
+    start_end : `dict` of `list` of `tuple`
+        dictionary containing code names and the start and stop indexes
+        of the code runs.
+    times : Numpy array of `float`
+        time array from the data file in hours
+
+    Returns
+    -------
+    elapsed : `dict` of `str` keys and `int` values
+        dictionary containing the code names and their associated elapsed
+        running time on slurm
+    """
+    # Finds time between two data points
+    dt = times[1] - times[0]
+    # Dict that finds number of times a certain code is mentioned in jobs
+    abrev_jobs = dict(Counter([word[:2] for lst in jobs for word in lst]))
```

Comment:
The `dict()` constructor is redundant here. From **collection** [docs](https://docs.python.org/3.6/library/collections.html#collections.Counter):
> Counter objects have a dictionary interface except that they return a zero count for missing items instead of raising a `KeyError`.